lua/mappings.lua

```diff
@@ -24,6 +24,12 @@ map("v", "x", [=[ "_x ]=], opt)
  this line too ]]
 --
 
+-- unix readline's keymap
```

Comment:
Maybe wrap this in a lua function and check if:
- if the `chadrc.lua` var does not exist, assume this functionality should be disabled (return, early exit the function)
- if the `chadrc.lua` var exists and is true, then run this block